packages/css/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "@markprompt/css",
+  "version": "0.0.0",
+  "description": "Common CSS for Markprompt components",
+  "license": "MIT",
+  "type": "module",
+  "exports": "./markprompt.css",
+  "main": "markprompt.css",
+  "files": [
+    "markprompt.css"
+  ],
```

It might be nice to add type definitions, so people can import it.

```suggestion
  "types": "./index.d.ts",
  "files": [
    "markprompt.css",
    "index.d.ts"
  ],
```

Or if we don’t care about TypeScript 4:

```suggestion
  "types": "./markprompt.d.css.ts",
  "files": [
    "markprompt.css",
    "markprompt.d.css.ts"
  ],
```

The type definitions just need to contain:

```ts
export {};
```